chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -101,7 +99,7 @@ public double KolmogorovProb(double z) {
      * @param option see above
      * @return the computed result
      */
-    public double KolmogorovTest(int na, double[] a, int nb, double[] b, String option) {
+    public double kolmogorovTest(int na, double[] a, int nb, double[] b, String option) {
```

Comment:
Codacy found an issue: [The method 'kolmogorovTest(int, double, int, double, String)' has an NPath complexity of 3280, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)